File: Python/FizzBuzz/fizzbuzz.py
i = 1
while i <= 15 :
    if i % 15 == 0 :
        print('FizzBuzz')
    elif i % 3 == 0 :
        print('Fizz')
    elif i % 5 == 0 :
        print('Buzz')
    else :
        print(i)
    i += 1

File: Pascal/FizzBuzz/fizzbuzz.pas
program FizzBuzz;

var i: Integer;

begin
   for i := 1 to 15 do
   begin
      if i mod 15 = 0 then
         writeln ('FizzBuzz')
      else if i mod 3 = 0 then
         writeln ('Fizz')
      else if i mod 5 = 0 then
         writeln ('Buzz')
      else
         writeln (i)
   end;
end.
